images/hub/jupyterhub_config.py

```diff
@@ -243,27 +243,6 @@
 
 c.Authenticator.enable_auth_state = get_config('auth.state.enabled', False)
 
-def generate_user_email(spawner):
-    """
-    Used as the EMAIL environment variable
-    """
-    return '{username}@{domain}'.format(
-        username=spawner.user.name, domain=email_domain
-    )
-
-def generate_user_name(spawner):
-    """
-    Used as GIT_AUTHOR_NAME and GIT_COMMITTER_NAME environment variables
-    """
-    return spawner.user.name
-
-c.KubeSpawner.environment = {
-    'EMAIL': generate_user_email,
```

Comment:
I wonder if someone is using the `EMAIL` env. var for other purposes. Perhaps we should make an entry to the CHANGELOG about to make sure they know whats to expect in between 0.7 and 0.8.

Reply:
#916